extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/OtTracerPropagator.java

```diff
@@ -58,7 +58,12 @@ public static OtTracerPropagator getInstance() {
     if (!spanContext.isValid()) {
       return;
     }
-    setter.set(carrier, TRACE_ID_HEADER, spanContext.getTraceIdAsHexString());
+    // OpenTracing trace id MUST be 64-bits therefore OpenTelemetry trace id is truncated to 64-bits
```

Comment:
```suggestion
    // Lightstep trace id MUST be 64-bits therefore OpenTelemetry trace id is truncated to 64-bits
```

This is specific to Lightstep I guess? Opentracing spec says 128-bit

https://github.com/opentracing/specification/blob/master/rfc/trace_identifiers.md#trace-context-http-headers